### PR TITLE
Add more dynamics, namely sfff and sfffz

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: https://musescore.org/en/node/359923#comment-1231431, https://musescore.org/en/node/92826#comment-1231428 and https://musescore.org/en/node/11888

### CODE REVIEW

**Review by rettinghaus — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
Could this one get reviewed please?

**Comment by Jojo-Schmitz** [comment]
Could this one get reviewed please?

**Comment by Jojo-Schmitz** [comment]
Could this one get reviewed please?

**Comment by rettinghaus** [comment]
Needs formatting and also tests fail.

**Comment by Jojo-Schmitz** [comment]
> Needs formatting and also tests fail.

Fixed

**Comment by zacjansheski** [comment]
Opcodes are in the dynamics / UI

Screenshot 2024-06-17 at 9 27 23 AM


Screenshot 2024-06-17 at 9 27 34 AM

File: src/engraving/dom/dynamic.cpp [hunk 1]
```diff
@@ -82,6 +82,10 @@ const std::vector<Dyn> Dynamic::DYN_LIST = {
     { DynamicType::SFF,     126, -18, true, "<sym>dynamicSforzando</sym><sym>dynamicForte</sym><sym>dynamicForte</sym>" },
     { DynamicType::SFFZ,    126, -18, true,
       "<sym>dynamicSforzando</sym><sym>dynamicForte</sym><sym>dynamicForte</sym><sym>dynamicZ</sym>" },
+    { DynamicType::SFFF,    127, -18, true,
+      "<sym>dynamicSforzando</sym><sym>dynamicForte</sym>sym>dynamicForte</sym><sym>dynamicForte</sym>" },
+    { DynamicType::SFFFZ,   127, -18, true,
+      "<sym>dynamicSforzando</sym><sym>dynamicForte</sym><sym>dynamicForte</sym>sym>dynamicForte</sym><sym>dynamicZ</sym>" },
```
Comment (Jojo-Schmitz) [inline src/engraving/dom/dynamic.cpp:88]: Ops, typo, missing <

**Comment by zacjansheski** [comment]
Please rebase (there was an issue with Windows builds, should be fixed now)

**Comment by Jojo-Schmitz** [comment]
done

**Comment by zacjansheski** [comment]
Tested on MacOS 14, Windows 11, Ubuntu 22.04.3. Approved

(once merged, sfff and sfffz will need to be added to this issue: https://github.com/musescore/MuseScore/issues/14736 )

**Comment by Jojo-Schmitz** [comment]
Did you forget to mark it as approved?

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**